<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsGeneral</class>
 <widget class="QWidget" name="SettingsGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>148</height>
   </rect>
  </property>
  <layout class="QFormLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="m_checkAutostart">
     <property name="text">
      <string>Launch %1 on operating system startup</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="m_checkForUpdatesOnStart">
     <property name="text">
      <string>Check for %1 updates on application startup</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="m_checkDisableDebugOutput">
     <property name="text">
      <string>Disable CLI debug output (but keep more serious warnings enabled)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
